packages/engine/Source/Core/Ellipsoid.js

```diff
@@ -254,6 +254,14 @@ Ellipsoid.MOON = Object.freeze(
   ),
 );
 
+/**
+ * An Ellipsoid instance initialized to a sphere with the mean radii of Mars.
```

Comment:
Should we say where we sourced our value from? For the earth WGS84 is well known and clear. Just trying to get ahead of someone asking